### DISCUSSION TITLE
Not update using other coluns

### DISCUSSION THREAD

**Discussion by Militao36** [prisma/prisma#6470] [Q&A] [answered]
I using prisma, not update using other coluns of table example.

Bottom of the code, other columns not visible from the table for updating. Just id.

```javascript
prisma.clientes.update({
  where: {
    id: 1
  },
  data: { }
})
```

The code below works, with other columns

```javascript
prisma.clientes.updateMany({
  where: {
    id: 1
  },
  data: {

  }
})
```

**Comment by ryands17** [comment, marked as answer]
@Militao36 👋 

`update` only accepts the primary key and unique columns specified in your `schema.prisma`. To update with more than one field that is not unique, you need to use `updateMany`. 

You can read more about all the methods [here](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#update).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
